spec/controllers/slack_controller_spec.rb

```diff
@@ -0,0 +1,153 @@
+require 'rails_helper'
+
+describe ::DiscourseSlack::SlackController do
+  routes { ::DiscourseSlack::Engine.routes }
```

Comment:
This is alittle odd, the routes are already drawn when we initialize the Rails app. Is there a reason we need to tell the tests to include the routes?